homeassistant/components/plum_lightpad/utils.py

```diff
@@ -0,0 +1,14 @@
+"""Reusable utilities for the Plum Lightpad component."""
+
+from plumlightpad import Plum
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+
+async def load_plum(username: str, password: str, hass: HomeAssistant) -> Plum:
```

Comment:
I like putting this code into it's own utility of some kind. Can we add a test here?

Reply:
sorry, I am having having troubles setting up my environment to be able to run tox - working through it, getting a dreadful "AttributeError: type object 'Callable' has no attribute '_abc_registry'" now. 
Happy to add more tests in follow-up PR when I figure it out. Meanwhile, if you have specific test in mind, can you add it to the PR? Thanks!